config/default.json

```diff
@@ -449,6 +449,9 @@
           }
         ]
       }
+    },
+    "redirect404": {
+      "notAllowedExtensions": [".png", ".gif", ".jpg", ".jpeg", ".woff", ".eot", ".woff2", ".ttf", ".svg", ".css", ".js", ".json", ".ico", ".tiff", ".tif", ".raw"]
     }
```

Comment:
This should be a subsection of “server” section called “server.customErrors.http404.notAllowedExtensions”

Reply:
Done